frontend/src/features/leases/detail/LeasePages/deposits/Deposits.test.tsx

```diff
@@ -69,6 +66,12 @@ const setup = (renderOptions: RenderOptions & { lease?: IFormLease } = {}): Rend
 };
 
 describe('Lease Deposits', () => {
+  beforeEach(() => {
```

Comment:
@asanchezr have to watch tests that take snapshots and use moment() or now.

Reply:
I see - good catch!